app/Http/Controllers/Api/GithubWebhookController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Services\GithubWebhookService;
+use Illuminate\Http\Request;
+
+class GithubWebhookController extends Controller
+{
+    protected $webhookService;
+
+    public function __construct()
+    {
+        $this->webhookService = new GithubWebhookService();
+    }
```

Comment:
```suggestion
    public function __construct(protected GithubWebhookService $webhookService)
    {
    }
```
https://stitcher.io/blog/constructor-promotion-in-php-8

Reply:
fixed constructors